### ISSUE TITLE
load comments button

### ISSUE THREAD

**Issue by ghost** [postmatic/epoch#152]
Load comments on demand, provide a button to load the comments and load X amount each time (or on scrolling event). This is the basic behavior of popular comment platforms
The plugin already loads the comments through ajax so its just a matter of adding a button and a couple of event handlers and offer the option for the admin.

**Comment by Shelob9** [comment]
This is a great idea:)

**Comment by Vernal** [comment]
Yes. Ideas we have plenty of! @natostanco stay tuned for 2.0. Lots of goodies coming there. Want to help out?

**Comment by jaredatch** [comment]
Just wanted to echo +1 on this feature request.

I have several clients that are looking to implement a similar feature.

Basically, they want N comments to show/load with the page, then have a button to load the remaining comments.

Best and cleanest example is on SimplyRecipes - http://www.simplyrecipes.com/recipes/ginger_honey_chicken_wings/

**Comment by jaredatch** [comment]
Might this be considered for 2.0 @Vernal ? Just curious :+1:

**Comment by Vernal** [comment]
<div></div><div>Might this be considered for 2.0 @Vernal ? Just curious :+1: 
<br/>
<br/>
<br/>---
<br/>Reply to this email directly or view it on GitHub:
<br/>https://github.com/postmatic/epoch/issues/152#issuecomment-186856347</div>
